packages/packages.json

```diff
@@ -55,6 +55,6 @@
         "skill/valory/abstract_round_abci/0.1.0": "bafybeigjddhk7epta7xpnfvv426xedff5abh4xlkwi6cqgp4vkutgkvydm",
         "skill/valory/transaction_settlement_abci/0.1.0": "bafybeifmgmwdkx4esemxjacjwzqkqymkuklb5nehkwqkx7v335fllgswcq",
         "skill/valory/termination_abci/0.1.0": "bafybeiea67epwwgngp7b3wavs6hpkaxv6etyaps6g6325bchfnf354mibq",
-        "skill/valory/mech_interact_abci/0.1.0": "bafybeib4vn6m2yumwoclh5aatcdt5yxcjc5owxmxy5o7t3nfzormgwkr64"
+        "skill/valory/mech_interact_abci/0.1.0": "bafybeibno654blw2bdnwgq7wi65mvd5jcug3jnggukjxszxnox3zvnwbf4"
```

Comment:
Not in the [IEKit release](https://github.com/valory-xyz/IEKit/blob/v0.14.9/packages/packages.json#L28) you referred to [here](https://github.com/valory-xyz/trader/pull/394#issuecomment-2643026651).